cmd/livepeer/livepeer.go

```diff
@@ -227,12 +228,15 @@ func main() {
 	}
 
 	// Setting config options based on specified network
-	var minGasPrice = big.NewInt(0)
 	if netw, ok := configOptions[*network]; ok {
 		if *ethController == "" {
 			*ethController = netw.ethController
-			minGasPrice = netw.minGasPrice
 		}
+
+		if !isFlagSet["minGasPrice"] {
+			*minGasPrice = netw.minGasPrice
```

Comment:
Moved this line out of the `*ethController` conditional block because we'd want to set the min gas price using the network config regardless of whether `-ethController` was specified or not.